ckanext/data_qld_theme/fanstatic/next-update-due.js

```diff
@@ -0,0 +1,36 @@
+jQuery(document).ready(function () {
+
+    UPDATE_FREQUENCY_DAYS = JSON.parse(jQuery('input[name="update_frequency_days"]').val())
```

Comment:
Should we set a default value of a empty dict just incase there is no value in the input field etc?